src/App.js

```diff
@@ -20,7 +24,8 @@ const App = () => {
       <MobileNavigationBar />
       <MobileNavigationBarBottom />
       <BodyWrapper>
-        <LandingPage />
+        {state.landingPage ? <LandingPage /> : false}
```

Comment:
there's a better way to do this.
```
{state.landingPage && <LandingPage />}
```

Reply:
Previously I was using && operator instead of using ternary. Then after reading the below article, I am following the practice to use the ternary operator
https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx